.github/workflows/release-binaries.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+on:
+  push:
+    tags:
+      # Push events to matching vX.X.X, i.e. v0.1.0
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+name: Create Binaries Release
+
+# Note: According to https://docs.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables,
+# the value of {{ github.ref }} is "refs/tags/<tag>", so we have to remove that prefix everywhere.
+jobs:
+  build-and-release:
+    name: Build and Release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.14'
+    - run: go version
```

Comment:
is the - in before run expected?

Reply:
Yes. This is technically a step without a name, that runs directly on the runner (i.e. not in a Docker container). See https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#choosing-a-runner.

Reply:
And this: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun

Reply:
The action step (`uses: actions/setup-go@v2`) is the equivalent of running docker run without an explicit entrypoint, it uses the action's default command (in this case it's a bunch of commands to download Go and configure the PATH). That's why it doesn't have a "run".